### PR TITLE
#34 manage ssl settings and add spec tests

### PR DESCRIPTION
**Pull request by your1p**
Work about #34 to manage ssl settings 

* Add parameters $ssl and $ssl_key and $ssl_cert to `init.pp` and template.
* Add relatives spec tests.

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -50,6 +50,11 @@
   # Users
   Optional[Hash] $users = undef,
 
+  # Ssl
+  Enum['enable','disable'] $ssl  = 'disable',
```
Comment (bastelfreak) [inline manifests/init.pp]: Datatypes \o/

File: templates/settings.json.epp [hunk 2]
```diff
@@ -22,20 +22,17 @@
...
-            "key"  : "/path-to-your/epl-server.key",
-            "cert" : "/path-to-your/epl-server.crt",
-            "ca": ["/path-to-your/epl-intermediate-cert1.crt", "/path-to-your/epl-intermediate-cert2.crt"]
+          "key"  : "<%= $etherpad::ssl_key -%>",
+          "cert" : "<%= $etherpad::ssl_cert -%>"
           },
-
-  */
+  <%- } else { -%>
+  // SSL module is not enable by default
```
Comment (bastelfreak) [inline templates/settings.json.epp]: can you fix the small typo here? I guess you mean `enabled`?

File: manifests/init.pp (hunk 1 above)
Comment (your1p) [reply manifests/init.pp]: What do you mean when you are talking about Datatypes ?

File: templates/settings.json.epp (hunk 2 above)
Comment (your1p) [reply templates/settings.json.epp]: Yes, thanks. I fixed it.

File: manifests/init.pp (hunk 1 above)
Comment (ekohl) [reply manifests/init.pp]: @bastelfreak is always happy to see PRs using datatypes.

**Pull request merged**